### PR TITLE
mame: Version bump; disable x86 as buildmaster runs out of RAM

### PR DESCRIPTION
**Pull request by PocketNerdIO**
Can someone please check that I've disabled the x86 build properly? Thank you!

### CODE REVIEW

File: games-emulation/mame/mame-0.264.recipe [hunk 1]
```diff
@@ -15,29 +15,21 @@ COPYRIGHT="1997-2023 MAMEDev and contributors"
...
 SOURCE_FILENAME="mame-$portVersion.tar.gz"
 SOURCE_DIR="mame-mame${portVersion/./}"
 PATCHES="mame-$portVersion.patchset"
 ADDITIONAL_FILES="
 	mame.sh
 	mame.rdef.in"
 
-ARCHITECTURES="all !x86_gcc2"
-SECONDARY_ARCHITECTURES="x86"
+ARCHITECTURES="all !x86_gcc2 !x86"
```
Comment (Begasus) [inline games-emulation/mame/mame-0.264.recipe:26]: You could still keep the secondary around and disable it there :)
```diff
-ARCHITECTURES="all !x86_gcc2 !x86"
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="!x86"
```
We don't provide primary x86, so it's no use declaring it there. :)

**Comment by PocketNerdIO** [comment]
Modified, as per @Begasus's suggestion. Thank you!

**Comment by Begasus** [comment]
@PocketNerdIO could you add a small notice why 32bit is disabled for buildmaster as per suggestiong from @OscarL ? Should be good to go.

**Comment by PocketNerdIO** [comment]
@OscarL @Begasus Sorry for not replying sooner. On it now.

> BTW, @PocketNerdIO... you _might_ want to double check your git settings, as the author on your commit doesn't matches your github user/email. If that's intentional, please disregard this section :-)

I hadn't realised Thank you for pointing it out - I've corrected my `.gitconfig`.

**Comment by Begasus** [comment]
Thanks!

**Comment by Begasus** [comment]
Build failure on buildmaster, patchset not included?
https://build.haiku-os.org/buildmaster/master/x86_64/logviewer.html?buildruns/10432/builds/81997.log

**Comment by jmairboeck** [comment]
The patchset is in the wrong directory, it needs to be in a "patches" subdirectory.

**Comment by PocketNerdIO** [comment]
Sorry! I had to do the push to GitHub from a different machine and clearly copied the patchset to the wrong place. I'm out right now, but I'll fix it this evening.

**Comment by PocketNerdIO** [comment]
Fixed in my branch. Do I need to create a new pull request?

**Comment by PocketNerdIO** [comment]
> (I, at least, never seen a PR being "reopened").

You make a fair point!

**Pull request merged**